powersimdata/design/transmission/mwmiles.py

```diff
@@ -51,20 +52,30 @@ def _calculate_mw_miles(original_grid, ct, exclude_branches=None):
     else:
         raise TypeError("exclude_branches must be None, list, tuple, or set")
 
-    base_branch = original_grid.branch
-    upgraded_branches = ct["branch"]["branch_id"]
-    for b, v in upgraded_branches.items():
+    base_branch_ids = set(original_grid.branch.index)
+    upgraded_branch_ids = set(ct["branch"]["branch_id"].keys())
+    transformed_branch = TransformGrid(original_grid, ct).get_grid().branch
+    if "new_branch" in ct:
+        upgraded_branch_ids |= set(transformed_branch.index)
```

Comment:
Am I reading this correctly - we could technically use `upgraded_branch_ids = set(transformed_branch.index)` to begin with, but in the case where there are no new branches, it's easier to just grab the scaled branch ids from the change table since there will be fewer to consider?

Reply:
I made a change when simplifying something here. We don't want the full transformed branch index, we want the entries in the change table plus any _new_ entries, which is the difference between the transformed branch and the base one. I've pushed a commit to fix.